packages/loading-skeleton/src/LoadingSkeleton/components/LoadingParagraph/LoadingParagraph.scss

```diff
@@ -0,0 +1,72 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/animation";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/border";
+
+.base {
+  background-color: $color-gray-300;
+  border-radius: calc(#{$border-solid-border-radius} * 3);
+  top: calc(#{$spacing-md} * 0.175);
+  margin-bottom: calc(#{$spacing-md} * 0.5);
+  height: calc(#{$spacing-md} * 0.625);
+}
+
+@keyframes fadeInOut {
+  0% {
+    opacity: 0.6;
+  }
+  50% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0.6;
+  }
+}
+
+.animated {
+  animation-name: fadeInOut;
+  animation-duration: calc(#{$animation-duration-deliberate} * 2);
+  animation-iteration-count: infinite;
+}
+
+.centered {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.reversed:not(.link) {
+  background-color: rgba($color-white-rgb, 0.1);
+}
+
+.link {
+  background-color: rgba($color-blue-500-rgb, 0.2);
+}
+
+.link.reversed {
+  background-color: rgba($color-white-rgb, 0.65);
+}
+
+.inline {
```

Comment:
As the stories are essentially our tests for this component via Chromatic, I think we should add an `inline` example story, as well as maybe something that is using `width` (or all in one?).

Reply:
I have created a new row within the sticker sheet to show inline. Do you think any other props need to be shown as well?

Reply:
One could argue, we would need a story representing every prop - possibly just for the sake of having them tested in Chromatic.
The ones I mentioned had the most value to me, where the others might be a bit "niche".

Reply:
That additional story is also using width so I've just updated the row header to specify it as well as inline

Reply:
I'd be pretty sure that the below is also a problem with the current component: when it wraps there's some left margin on the first item of the 2nd row :(
I wouldn't worry about it right now, but I think it should probably be put in a flex container instead. If we only had one handy.... :)

<img width="402" alt="image" src="https://user-images.githubusercontent.com/763385/159833145-2cf01946-837f-4e82-ba7d-7c3e15a9f66f.png">